src/angular/hq/src/app/staff-dashboard/staff-dashboard.component.html

```diff
@@ -149,114 +182,305 @@
-      >
-        <div>{{ date.date | date: "EEEE" }} | {{ date.date }}</div>
-        <div>{{ date.totalHours | number: "1.2-2" }} hours</div>
-      </div>
-    }
-    <table class="w-full border-spacing-0 border-separate table-fixed">
-      <thead>
-        <tr class="top-0 text-left">
-          <th
-            scope="col"
-            colspan="2"
-            style="width: 28px"
-            class="bg-blue-900 border-steel-blue-600 border-y py-3 pr-2 text-nowrap"
-          ></th>
-          <th
-            style="width: 120px"
-            class="bg-blue-900 border-steel-blue-600 border-y py-3 pl-3 text-nowrap"
-          >
-            Hrs
-          </th>
-          <th
-            style="width: 160px"
-            class="bg-blue-900 border-steel-blue-600 border-y py-3 pl-2"
-          >
-            Date
-          </th>
-          <th
-            style="width: 160px"
-            class="bg-blue-900 border-steel-blue-600 border-y py-3 pl-5 text-nowrap"
-          >
-            Chrg Code
-          </th>
-          <th
-            style="width: 160px"
-            class="bg-blue-900 border-steel-blue-600 border-y py-3 pl-2"
-          >
-            Client
-          </th>
-          <th
-            style="width: 160px"
-            class="bg-blue-900 border-steel-blue-600 border-y py-3 pl-2"
-          >
-            Project
-          </th>
-          <th
-            style="width: 160px"
-            class="bg-blue-900 border-steel-blue-600 border-y py-3 pl-2 text-nowrap"
+<div class="w-full h-screen">
+  <!-- @if (
+    staffDashboardService.period.value === Period.Week ||
+    (staffDashboardService.showAllRejectedTimes$ | async)
+  ) { -->
+  <div
+    class="bg-black-alt"
+    [hidden]="
+      staffDashboardService.period.value === Period.Today &&
+      (staffDashboardService.showAllRejectedTimes$ | async) !== true
+    "
+  >
+    @if (staffDashboardService.time$ | async; as dashboard) {
+      @for (date of dashboard.dates; track date.date) {
+        @if (dashboard.dates.length > 1) {
+          <div
+            class="px-5 pb-1 pt-4 font-open-sans text-nav-disabled text-xs font-bold uppercase flex justify-between"
           >
-            Activity / Task
-          </th>
-          <th class="bg-blue-900 border-steel-blue-600 border-y py-3 pl-2">
-            Description
-          </th>
-          <th
-            style="width: 40px"
-            class="bg-blue-900 border-steel-blue-600 border-y"
-          ></th>
-        </tr>
-      </thead>
-      <tbody>
-        @if (
-          (staffDashboardService.showAllRejectedTimes$ | async) === false &&
-          date.canCreateTime
-        ) {
-          <tr
-            hq-staff-dashboard-time-entry
-            (hqTimeChange)="upsertTime($event)"
-            [time]="{ date: date.date, timeStatus: timeStatus.Unsubmitted }"
-          ></tr>
+            <div>{{ date.date | date: "EEEE" }} | {{ date.date }}</div>
```

Comment:
![image](https://github.com/user-attachments/assets/fdbbaaec-484e-4cca-bd7c-db13334f9a05)

This should match the current date we are filtering time to.